EIPS/eip-5548.md

```diff
@@ -0,0 +1,126 @@
+---
+eip: 5548
+title: NFT Operator Approval Control
+description: An EIP-721 extension to deny specific smart contracts the ability to spend tokens on an owner's behalf
+author: Mitchell F Chan (@mitchellfchan), Harm van den Dorpel (@harmvandendorpel), et al.
```

Comment:
> et al.
[Editorial] Are there other authors? Usually all authors are listed unless there are particular reasons.?

Reply:
Adding more authors as they contribute. Used "et al." for now as a placeholder, anticipating more coming... which they are, thankfully!